src/store/story.js

```diff
@@ -65,6 +78,25 @@ export default {
     setImg: ({commit}, img) => {
       commit('setImg', img)
     },
+    setImgTitle: ({commit}, title) => {
+      commit('setImgTitle', title)
+    },
+    fetchImgAttribution: async ({commit}, image) => {
+      const url = `https://${wikiSubdomain}.wikipedia.org/w/api.php?format=json&formatversion=2&origin=*&action=query&prop=imageinfo&iiextmetadatafilter=License%7CLicenseShortName%7CImageDescription%7CArtist&iiextmetadatalanguage=en&iiextmetadatamultilang=1&iiprop=url%7Cextmetadata&titles=${encodeURIComponent(image.title)}`
+      const rawAttribution = await (await fetch(url)).text()
+      const parsedAttribution = JSON.parse(rawAttribution)
+      const imageInfo = parsedAttribution.query.pages[ 0 ].imageinfo[0]
+      if (imageInfo) {
+        const { Artist, LicenseShortName } = imageInfo.extmetadata
+        const attribution = {
+          author: Artist ? strip(Artist.value) : 'Unknown author',
```

Comment:
`i18n()`

Reply:
How would it look if we don't know anything when the author is unknown?

Reply:
I'm wondering if we could omit the author string completely when the author is unknown instead of showing the literal "Unknown author" string.

Reply:
Oh gotcha, yes that would be another easy approach with a `v-if`. The "Author unknown" string is consistent with the design of previous galleries we've done, I suppose we could check with the team in the phab if there's a preference